src/components/Loading.jsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
 import {Container, Row, Card} from 'react-bootstrap';
 
-const Loading = () => {
+const Loading = ({loadingMessage}) => {
```

Comment:
one last nitpick, but you should have an empty string by default `loadingMessage = ''` otherwise, if you don't pass in a `loadingMssage` your UI will show `Loading undefined`